libtransmission/peer-mgr.cc

```diff
@@ -2081,7 +2080,7 @@ void enforceSwarmPeerLimit(tr_swarm* swarm, size_t max)
     }
 
     // close all but the `max` most active
-    auto peers = std::vector<tr_peerMsgs*>{ n - max };
```

Comment:
ugh.

Reply:
Well actually this works as-is, but the brace syntax combined with this overload of `std::vector`'s constructor is dangerous.